src/modules/release/utils.ts

```diff
@@ -44,10 +44,10 @@ export class ReleaseUtils {
     return version
   }
 
-  public incrementVersion = (rawOldVersion: string, releaseType: string, tagName: string) => {
+  public incrementVersion = (rawOldVersion: string, releaseType: semver.ReleaseType, tagName: string) => {
     const oldVersion = semver.valid(rawOldVersion, true)
     if (tagName !== 'stable' && releaseType !== 'prerelease') {
-      return semver.inc(oldVersion, `pre${releaseType}`, tagName)
+      return semver.inc(oldVersion, `pre${releaseType}` as semver.ReleaseType, false ,tagName)
```

Comment:
Were you able to test this? I think a `vtex release` on an app would be sufficient